Makefile

```diff
@@ -0,0 +1,13 @@
+venv: venv/bin/activate
+
+venv/bin/activate: requirements.txt
+	if [ ! -d venv ]; then \
+	    virtualenv --no-setuptools venv; \
+	fi
+	. venv/bin/activate; pip install -r requirements.txt
+
+clean:
+	$(RM) -rf venv
+	find . -name "*.pyc" -exec $(RM) -rf {} \;
+
+.PHONY:clean venv
```

Comment:
i was thinking about doing this at one point, and then got distracted on other projects. very clean change.